ansible/templates/cluster-template.yaml.j2

```diff
@@ -85,12 +85,7 @@ apiEndpoints:
 #    # Network ranges of sources you'd like Kubernetes API accesses to be allowed from, in CIDR notation. Defaults to ["0.0.0.0/0"] which allows any sources.
 #    # Explicitly set to an empty array to completely disable it.
 #    # If you do that, probably you would like to set securityGroupIds to provide this load balancer an existing SG with a Kubernetes API access allowed from specific ranges.
-    apiAccessAllowedSourceCIDRs:
```

Comment:
I moved those to the SG from the AWS account and are dynamically conducted. As in here: https://github.com/Financial-Times/aws-composer-account-content-platform-prod/blob/master/cloudformation/k8s/api-internal-access.yaml#L31